package-lock.json

```diff
@@ -22334,7 +22334,7 @@
     },
     "packages/ui": {
       "name": "@lion/ui",
-      "version": "0.1.5",
```

Comment:
this change shouldn't be part of this PR, I assume.

Reply:
Ran an `npm i` and quickly also updated the package-lock.
I also added it as (2) on the description above, so we can include this in this PR as well, if you agree.

Reply:
sure, i dont know how strict you are with these things. Fine by me